app/src/main/java/com/gkp/tasky/nagivation/TaskyNavHost.kt

```diff
@@ -0,0 +1,27 @@
+package com.gkp.tasky.nagivation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.rememberNavController
+import com.gkp.auth.presentation.login.navigation.LoginRoute
+import com.gkp.auth.presentation.login.navigation.loginScreen
+import com.gkp.auth.presentation.register.navigation.navigateToRegister
+import com.gkp.auth.presentation.register.navigation.registerScreen
+
+@Composable
+fun TaskyNavHost() {
+    val navController = rememberNavController()
+    NavHost(
+        startDestination = LoginRoute,
+        navController = navController
+    ) {
+        loginScreen(
+            onLoginButtonClick = {},
+            onSignUpTextClick = { navController.navigateToRegister()}
+        )
+        registerScreen(
+            onGetStartedButtonClick = {},
+            onFabBackClick = {navController.popBackStack()}
```

Comment:
For user-initiated back stack popping, I'd prefer `navigateUp`, since that is aware of your back stack and will only navigate to the previous screen (and not generally just pop one from the back stack whatever it is).
When this is done by `popBackStack`, clicking a button and there's a transition animation or so, it can happen that the user could press the button multiple times when tapped fast. That would clear more of the back stack as you'd want here

Reply:
Ok. I will make the change accordingly